src/EDGAR/parser.py

```diff
@@ -340,7 +339,11 @@ def get_annotated_submissions(self, tikr,
 
                     out.append(submission)
             elif form_type == '8k' and document_type == '8k':
-                raise RuntimeWarning("FORM 8K UNIMPLEMENTED FOR SUBMISSION")
+                if self._is_8k_annotated(
```

Comment:
We should not make a function for every form that exists (There are hundreds more).
Let's just make a generic ```_contains_annotations(tikr: str, submission: str, silent=False)```
Is the ```8k``` function different in any way from the ```10q``` besides the name?